pkg/templates/untrustedimages/template.go

```diff
@@ -0,0 +1,55 @@
+package untrustedimages
+
+import (
+	"fmt"
+	"regexp"
+
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/untrustedimages/internal/params"
+	"golang.stackrox.io/kube-linter/pkg/templates/util"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	templateKey = "untrusted-images"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "Untrusted images",
+		Key:         templateKey,
+		Description: "Flag applications running containers with untrusted container images",
+		SupportedObjectKinds: config.ObjectKindsDesc{
+			ObjectKinds: []string{objectkinds.DeploymentLike},
+		},
+		Parameters:             params.ParamDescs,
+		ParseAndValidateParams: params.ParseAndValidate,
+		Instantiate: params.WrapInstantiateFunc(func(p params.Params) (check.Func, error) {
+			return util.PerContainerCheck(func(container *v1.Container) []diagnostic.Diagnostic {
+
+				// checking whether the container image is a trusted or not
+
```

Comment:
I would just remove the comment and these blank lines, I don't think the comment adds any extra information.
```suggestion
```

Reply:
removed the comment & lines.